Dockerfile

```
@@ -22,5 +22,4 @@ RUN pip install --no-cache-dir -r requirements.txt
 ENV PYTHONPATH=/app
 
 # Command to run the FastAPI server
-CMD python aas_repository_semantic_matcher/matcher.py "$REPOSITORY_ENDPOINT"
-
+CMD python aas_repository_semantic_matcher/matcher.py "$REPOSITORY_ENDPOINT"
```

```suggestion
CMD python aas_repository_semantic_matcher/matcher.py "$REPOSITORY_ENDPOINT"

```

Please reintruduce the empty line at the end of the file. (See [POSIX definition of a line](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206).) Some older unix tools and some git diff tools may have problems if that new line character is missing.